contract/src/main.rs

```diff
@@ -295,12 +323,12 @@ pub extern "C" fn init() {
     );
     runtime::put_key(RECEIPT_NAME, storage::new_uref(receipt_name).into());
     runtime::put_key(
-        &format!("{}{}", CEP78_PREFIX, collection_name),
+        &format!("{CEP78_PREFIX}{collection_name}"),
         storage::new_uref(package_hash).into(),
     );
     runtime::put_key(
         NFT_METADATA_KIND,
-        storage::new_uref(nft_metadata_kind as u8).into(),
+        storage::new_uref(nft_metadata_kinds).into(),
```

Comment:
This is also breaking, we should preserve the original `u8` write here, but then add a new named key entry for the full map